arch/src/aarch64/fdt.rs

```diff
@@ -192,32 +189,9 @@ fn create_cpu_nodes(
         fdt.end_node(cpu_node)?;
     }
 
-    // If there is a valid cpu topology config, create the cpu-map node.
-    if (threads_per_core > 0)
-        && (cores_per_package > 0)
-        && (packages > 0)
-        && (num_cpus as u8 == threads_per_core * cores_per_package * packages)
```

Comment:
This is unnecessary too. For FDT, VCPU hotplug is impossible, booting VCPU number is identical as max VCPU number. While there is check between the value of `threads_per_core * cores_per_package * package` and max VCPU number. So this is always guaranteed.